### PR TITLE
feat(app): build password generator 

### PR DESCRIPTION
**Pull request by ellaboevans**
This pull request contains the solution for the password generator app lab.

### CODE REVIEW

File: week_3/password-generator-app/main.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+checkboxes.forEach((checkbox) =>
+  checkbox.addEventListener("change", updateButtonState)
+);
+
+document.addEventListener("DOMContentLoaded", () => {
+  passwordDisplay.textContent = "P4$5W0rD!";
+  passwordDisplay.style.color = "#817d92";
+});
+
+// Functions
```
Comment (fredrickyalley) [inline week_3/password-generator-app/main.js:37]: this comment isnt necessary

File: week_3/password-generator-app/main.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+const copiedText = document.getElementById("copied");
+const slider = document.getElementById("slider");
+const countDisplay = document.getElementById("count");
+const upperCase = document.getElementById("uppercase");
+const lowerCase = document.getElementById("lowercase");
+const numbers = document.getElementById("numbers");
+const symbols = document.getElementById("symbols");
+const checkboxes = document.querySelectorAll('input[type="checkbox"]');
+
+// Instance of PasswordGenerator
```
Comment (fredrickyalley) [inline week_3/password-generator-app/main.js:18]: this comment too should be removed

File: week_3/password-generator-app/main.js (hunk 1 above)
Comment (ellaboevans) [reply week_3/password-generator-app/main.js:37]: Sure..

File: week_3/password-generator-app/main.js (hunk 2 above)
Comment (ellaboevans) [reply week_3/password-generator-app/main.js:18]: Yes please.

File: week_3/password-generator-app/main.js (hunk 1 above)
Comment (ellaboevans) [reply week_3/password-generator-app/main.js:37]: it has been removed.

File: week_3/password-generator-app/main.js (hunk 2 above)
Comment (ellaboevans) [reply week_3/password-generator-app/main.js:18]: The comment has been taken off

**Review by fredrickyalley — APPROVED** [review]

**Pull request merged**